packages/@aws-cdk/cloud-assembly-schema/scripts/schema.js

```diff
@@ -0,0 +1,129 @@
+const jsonpatch = require('fast-json-patch');
+const TJS = require('typescript-json-schema');
+const path = require('path');
+const semver = require('semver');
+
+function bump() {
+
+  const metadataPath = '../schema/cloud-assembly.metadata.json';
+
+  var metadata = require(metadataPath);
+
+  const oldVersion = metadata.version
+  const newVersion = semver.inc(oldVersion, 'major');
+
+  console.log(`Updating schema version: ${oldVersion} -> ${newVersion}`);
+
+  const patched = jsonpatch.applyPatch(metadata,
+    [
+      {
+        op:"replace",
+        path: "/version",
+        value: newVersion
+      }
+    ]
+  ).newDocument;
+
+  const out = path.join(__dirname, metadataPath);
+  fs.writeFileSync(out, JSON.stringify(patched, null, 4));
+
+}
+
+function generate(out, shouldBump) {
+
+  const settings = {
+    required: true,
+    ref: true,
+    topRef: true,
+    noExtraProps: true,
+    out: out
+  };
+
+  const compilerOptions = {
+    strictNullChecks: true
+  };
+
+  const program = TJS.getProgramFromFiles([path.join(__dirname, "../lib/manifest.d.ts")], compilerOptions);
+  const schema = TJS.generateSchema(program, 'AssemblyManifest', settings);
+
+  const patches = patchDefaults(schema);
+
+  const addAnyOfAny = {
+    op: 'add',
+    path: "/definitions/MetadataEntry/properties/data/anyOf/0",
+    value: {
+      "description": "Any form of data. (for backwards compatibility to version < 1.31.0)"
+    },
+  };
+
+  patches.push(addAnyOfAny);
+
+  const patched = jsonpatch.applyPatch(schema, patches).newDocument;
```

Comment:
got to say that this also seems like an overkill and an extra unneeded dependency on `jsonpatch`. 

Can't you just do something like this?

```js
schema.definitions.MetadataEntry.properties.data.anyOf.push({ description: 'foo' });
```

This will uglily (and justifiably) fail if this entry is invalid

Reply:
👍